### ISSUE TITLE
starTreeLikelihood XML element does not parse

### ISSUE THREAD

**Issue by GuyBaele** [beast-dev/beast-mcmc#1006]
In the examples folder, the testStarTreeTest.xml file contains a starTreeLikelihood XML element:
```
<starTreeLikelihood id="treeLikelihood">
		<patterns idref="patterns"/>
		<treeModel idref="treeModel"/>
		<siteModel idref="siteModel"/>
</starTreeLikelihood>
```
However, there is no parser in the BEAST code base for such an XML element. Was there at some point or is this simply an invalid XML file and should we remove it?

**Comment by rambaut** [comment]
This looks like old and redundant stuff. I vote that we remove the XML. Basically 1.10 doesn't support StarBEAST (as this is now replaced by StarBEAST 2 and 1.8.4 can be used for legacy XML).

**Comment by GuyBaele** [comment]
Agreed.

**Comment by rambaut** [comment]
In fact we could replace StarBEAST parsers with a message to say use 1.8.4 or move to StarBEAST2?

**Comment by msuchard** [comment]
Probably should advise to "use 1.8.4" as I am not sure that StarBEAST2 uses the same XML specification.

**Comment by alexeid** [comment]
We should advice new users interested in StarBEAST-like models to use BEAST 2.5 and only use 1.8.4 for legacy code. 

StarBEAST2 is way more computationally efficient than 1.8.4, and also allows:

* relaxed clock on the species tree instead of just the gene trees
* analytical integration of population sizes.
* improved MCMC operators

There have been 4-5 years of continuous development on StarBEAST-like models in BEAST2 by four different research groups. Related models implemented in BEAST 2.5 include

* Species Networks
* IM-like models with full species tree estimation
* Species delimitation models.

**Comment by rambaut** [comment]
But in the mean time, delete the XMLs and close the issue. (one of them is called 'BrithDeathSingleLocus' anyway).
